src/notation/view/abstractnotationpaintview.cpp

```diff
@@ -560,7 +560,6 @@ void AbstractNotationPaintView::hideElementPopup()
 void AbstractNotationPaintView::toggleElementPopup(const ElementType& elementType, const RectF& elementRect)
 {
     if (m_isPopupOpen) {
-        hideElementPopup();
```

Comment:
This may also have unintended effects, and makes the method not do what it promises. If you only want to _open_ the popup, just call `showElementPopup`; `toggleElementPopup` should _toggle_ the popup.